lib/diagnostic.rb

```diff
@@ -45,12 +55,12 @@
 # What is the type of object in Ruby for decimal numbers? Write your answer in
 # a Ruby string.
 
-Diagnostic.decimal = 'your response here'
+Diagnostic.decimal = 'float'
 
 # What is the type of object in Ruby for integer numbers? Write your answer in
 # a Ruby string.
 
-Diagnostic.integer = 'your response here'
+Diagnostic.integer = 'interger'
 
 # Write an example of a decimal and an integer in Ruby. Replace the Ruby strings
 # in the following array with your examples.
```

Comment:
numbers - `[1.0, 1]`
falsy - `[nil, false]`
flow_control - `"The Dark Night"`
else_if - 'elsif'
returns - `false`
remove_from_array - `arr.pop 2`
array_add_to - `[12, 34, 56, 67, nil, nil, 9]`
person_hash - person = `{first_name: "Natalia", age: 88}`
hash_default - `person.default = \"\#{person[:first_name]}`